localisation/wc_religions.csv

```diff
@@ -94,157 +94,159 @@ arakkoa_religion_group;Arakkoa;Arakkoa;Arakkoa;;;;;;;;;;;x
+DRUID;Druid;Druide;Druide;;;;;;;;;;;x
+pandarian_beliefs;Pandaren;Pandar�ne;Pandaren;;;;;;;;;;;x
+august_celestials;Celestial;V�n�rable;Himmlisch;;;;;;;;;;;x
+GOD_AUGUST_CELESTIALS;the August Celestials;les Astres V�n�rables;die himmlischen Erhabenen;;;;;;;;;;;x
+SHA;the Sha;les Sha;die Sha;;;;;;;;;;;x
+jinyu_beliefs;Jinyu;jinyue;Jinyu;;;;;;;;;;;x
+WATERSPEAKER;Waterspeaker;Eauracle;Wassersprecher;;;;;;;;;;;x
+THE_WATER;the Water;l'Eau;das Wasser;;;;;;;;;;;x
+THE_RIVER;the River;la Rivi�re;der Fluss;;;;;;;;;;;x
+THE_SEA;the Sea;la Mer;die See;;;;;;;;;;;x
+THE_DROUGHT;the Drought;;ie D�rre;;;;;;;;;;;x
 august_celestials_DESC;The August Celestials are four wild gods who defend the temples from the evils of the sha and mogu. The celestials built temples to train monks in their respective styles after the pandaren revolution. Many native races of Pandaria respect these creatures and worship them as gods.;;;;;;;;;;;;;x
 jinyu_beliefs_DESC;Water speaking is a belief in Jinyu society. It allows their spiritual leaders to speak and listen to the waters that flow throughout Pandaria, making them extremely wise and terribly powerful.;;;;;;;;;;;;;x
 vrykul_religion_group;Vrykultruar;vrykultruare;;;;;;;;;;;;x
-odyn;Odyn;;;;;;;;;;;;;x
-GOD_ODYN;Odyn;Odyn;;;;;;;;;;;;x
-GOD_HELYA;Helya;Helya;;;;;;;;;;;;x
-odyn_DESC;Originally created alongside the Dragon Aspects with the task of protecting Azeroth, the Valarjar were judged the mightiest titan-forged warriors by Odyn. It was them who he thought were the only ones worthy of protecting Azeroth. When, instead of his Valarjar, the Dragon Aspects were given the powers of the Titans to safeguard Azeroth after their departure, Helya alongside Odyn raised a section of Ulduar up into the skies to create the Halls of Valor, a new home for his Valarjar warriors. To this day, the strongest Vrykul are hand-picked by Odyn and have their spirits lifted by the Val'kyr to the Halls where they will fight on eternal.; this section is now known as the Halls of Valor. Vrykul, or other warriors who prove their courage and tenacity, hope one day to walk amongst the Valarjar and Val'kyr within Odyn's Halls of Valor.;;;;;;;;;;;;x
-helya;Helya;;;;;;;;;;;;;x
+odyn;Odyn;Odyn;Odyn;;;;;;;;;;;x
+GOD_ODYN;Odyn;Odyn;Odyn;;;;;;;;;;;x
+GOD_HELYA;Helya;Helya;Helya;;;;;;;;;;;x
+odyn_DESC;Originally created alongside the Dragon Aspects with the task of protecting Azeroth, the Valarjar were judged the mightiest titan-forged warriors by Odyn. It was them who he thought were the only ones worthy of protecting Azeroth. When, instead of his Valarjar, the Dragon Aspects were given the powers of the Titans to safeguard Azeroth after their departure, Helya alongside Odyn raised a section of Ulduar up into the skies to create the Halls of Valor, a new home for his Valarjar warriors. To this day, the strongest Vrykul are hand-picked by Odyn and have their spirits lifted by the Val'kyr to the Halls where they will fight on eternal.;;;;;;;;;;;;;x
```

Comment:
```suggestion
odyn_DESC;Originally created alongside the Dragon Aspects with the task of protecting Azeroth, the Valarjar were judged the mightiest titan-forged warriors by Odyn. It was them who he thought were the only ones worthy of protecting Azeroth. When, instead of his Valarjar, the Dragon Aspects were given the powers of the Titans to safeguard Azeroth after their departure, Helya alongside Odyn raised a section of Ulduar up into the skies to create the Halls of Valor, a new home for his Valarjar warriors. To this day, the strongest Vrykul are hand-picked by Odyn and have their spirits lifted by the Val'kyr to the Halls where they will fight on for eternity.;;;;;;;;;;;;;x
```

Reply:
done